### ISSUE TITLE
`vrms-rpm` lacks the ability to report full names of fake packages with GPG keys

### ISSUE THREAD

**Issue by Ricky-Tigg** [suve/vrms-rpm#89]
Hello. In v. 2.2 **fake packages with GPG keys** are taken in account in the query. That's also what the rpm utility does when `rpm -qa` is invoked. In a system where such packages can be found, their presence can be attested with `vrms-rpm --list all | grep ' gpg-pubkey$'`. A preferable default behaviour would be to not take them in account. As noticeable, `vrms-rpm` lacks the ability to report **full names** of such packages.
```
$ vrms-rpm --list all | grep ' gpg-pubkey$'
 - gpg-pubkey
 - gpg-pubkey
$ rpm -qa | grep '^gpg-pubkey-*-*'
gpg-pubkey-5323552a-6112bcdc
gpg-pubkey-ba3c3a2c-5eb88cc6
```

**Comment by suve** [comment]
This is _kinda_ fixed in the current trunk version. `gpg-pubkey` packages store the key hashes in the Version and Release fields, and some time ago we added code where packages with duplicate names are printed in a `name-epoch:version-release` fashion to allow to distinguish between them.

One alternative would be to treat `gpg-pubkey` packages as a special case that always needs to be printed in a `name-version-release` fashion.

**Comment by Ricky-Tigg** [comment]
"_`gpg-pubkey` packages store the key hashes in the Version and Release fields_" | I did not know that. Hashes probably truncated.
```
$ rpm -qi $(rpm -qa gpg-pubkey | head -1) | sed -n '2,3p'
Version     : 5323552a
Release     : 6112bcdc
```
"_to treat `gpg-pubkey` packages as a special case_" | a special case which they are. Seems pertinent.

**Comment by suve** [comment]
This should be fixed by 9716cfe45f962813efd8eff5dfef4aa536cc1c25, which makes the program always print version-release information if a package seems to be a pubkey package. The two criteria used are:
- Name is `gpg-pubkey` (duh)
- Architecture is `​` (i.e. none / unset)
